Clients/src/presentation/pages/Assessment/index.tsx

```diff
@@ -1,5 +1,72 @@
+import Button from "@mui/material/Button";
+import Stack from "@mui/material/Stack";
+import { Paper, styled, Typography, useTheme } from "@mui/material";
+
+const Item = styled(Paper)(({ theme }) => ({
+  backgroundColor: '#f9fafc',
+  ...theme.typography.body2,
+  padding: theme.spacing(1),
+  border:"1px solid #EAECF0",
+  boxShadow: "none",
+  paddingRight: "150px",
+  paddingLeft: "25px",
+  paddingTop: "10px",
+  paddingBottom: "10px",
+}));
+
 const Assessment = () => {
```

Comment:
In many parts, we have hard-coded values here.
Let's make it more flexible and use the theme file.
Use now, in parts that it is possible, you can use:
`theme. ...`
And replace the corresponding values.